src/plugins/scriptedwizard/resources/wxwidgets/wizard.script

```diff
@@ -1153,18 +1160,100 @@ function OnGetPrevPage_WxAddLib()
 
 function OnEnter_WxAddLib(fwd)
 {
+    if (fwd)
+    {
+        if (IsDLL) // no linking against any winsock needed
+        {
+            Wizard.CheckCheckbox(_T("chkWxLinkWinSock2"), false);
+            Wizard.EnableWindow(_T("chkWxLinkWinSock2"), false);
+        }
+        else
+        {
+            Wizard.CheckCheckbox(_T("chkWxLinkWinSock2"), IntToBool(ConfigManager.Read(_T("/wx_project_wizard/winsock2"), 0)));
```

Comment:
This is not `ConfigManager` instead of `GetConfigManager`. Is this correct? It probably works, but it is different from the other code.

Reply:
Yes, this is a mistake. Thanks for noticing.

Reply:
GetConfigManager() should be better. Technically both work, but the GetConfigManager() variant is more similar to how this is done in C++, so I'd prefer if it is used instead.